packages/openneuro-app/src/scripts/dataset/files/viewers/file-viewer-nifti.jsx

```diff
@@ -1,19 +1,26 @@
-import React from 'react'
+import React, { useEffect, useRef } from 'react'
 import PropTypes from 'prop-types'
-import Papaya from '../../../common/partials/papaya.jsx'
-import styled from '@emotion/styled'
+import { Niivue } from '@niivue/niivue'
 
-const StyleWrapper = styled.div`
-  div.papaya-wrap {
-    margin: auto;
-  }
-`
+const FileViewerNifti = ({ imageUrl }) => {
+  const canvas = useRef()
+  useEffect(() => {
+    const volumeList = [
+      {
+        url: imageUrl,
+        volume: { hdr: null, img: null },
+        colorMap: 'gray',
+        opacity: 1,
+        visible: true,
+      },
+    ]
+    const nv = new Niivue()
```

Comment:
currently, drag and drop is enabled by default in niivue. You may want to disable that so that users do not accidentally drag and drop a file or url onto the canvas. 

you can modify this behaviour by supplying an options object when you initialise niivue. 

something like:

```
const nv = new Niivue({dragAndDropEnabled: false})
```

you can see a list of all options available here: https://niivue.github.io/niivue/devdocs/niivue.js.html#line82